src/sentry/tasks/deletion.py

```diff
@@ -267,6 +267,41 @@ def delete_group(object_id, transaction_id=None, **kwargs):
         )
 
 
+@instrumented_task(
+    name='sentry.tasks.deletion.delete_groups',
+    queue='cleanup',
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES
+)
+@retry(exclude=(DeleteAborted, ))
+def delete_groups(object_ids, transaction_id=None, eventstream_state=None, **kwargs):
+    from sentry import deletions, eventstream
+    from sentry.models import Group
+
+    max_batch_size = 100
+    current_batch, rest = object_ids[:max_batch_size], object_ids[max_batch_size:]
+
+    task = deletions.get(
+        model=Group,
+        query={
+            'id__in': current_batch,
+        },
+        transaction_id=transaction_id or uuid4().hex,
```

Comment:
Let's set this `uuid4().hex` outside of this block so that we can pass the same identifier down if we get to `has_more`.

Reply:
Good catch